<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FireAlarm</class>
 <widget class="QWidget" name="FireAlarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
    font: 20px &quot;微软雅黑&quot;;
}

QWidget
{
	background-color: black;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 36, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 36, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 36, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 36, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 36, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 36, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>70</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 36, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>70</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 36, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblfireAlarmA1">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>190</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/fireAlarmOk.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblfireAlarmB1">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>190</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/fireAlarmOk.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblfireAlarmC1">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>190</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/fireAlarmOk.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblfireAlarmD1">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>190</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/fireAlarmOk.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblfireAlarmD2">
   <property name="geometry">
    <rect>
     <x>545</x>
     <y>190</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/fireAlarmOk.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblfireAlarmC2">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>190</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/fireAlarmOk.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblfireAlarmB2">
   <property name="geometry">
    <rect>
     <x>785</x>
     <y>190</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/fireAlarmOk.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblfireAlarmA2">
   <property name="geometry">
    <rect>
     <x>905</x>
     <y>190</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/fireAlarmOk.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/fireAlarmOk.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/fireAlarmWarning.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/fireAlarmError.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>480</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/fireAlarmOffLine.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
font: 16px, &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>正常</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>540</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
font: 16px, &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>火警</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>540</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
font: 16px, &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>故障</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>540</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
font: 16px, &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>通讯故障</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetting">
   <property name="geometry">
    <rect>
     <x>824</x>
     <y>330</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    font: 18px &quot;微软雅黑&quot;;
    color: white;
	background-color: black;
	border:1px solid white;
}</string>
   </property>
   <property name="text">
    <string>整车火灾重启</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblVehicleNumberA1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
font: 18px, &quot;微软雅黑&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>A001</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVehicleNumberB1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
font: 18px, &quot;微软雅黑&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>A001</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVehicleNumberC1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
font: 18px, &quot;微软雅黑&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>A001</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVehicleNumberD1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
font: 18px, &quot;微软雅黑&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>A001</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVehicleNumberD2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
font: 18px, &quot;微软雅黑&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>A001</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVehicleNumberC2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
font: 18px, &quot;微软雅黑&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>A001</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVehicleNumberB2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
font: 18px, &quot;微软雅黑&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>A001</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVehicleNumberA2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
font: 18px, &quot;微软雅黑&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>A001</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTip">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>791</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px &quot;微软雅黑&quot;;
background-color: black;
border: 0px;
color: red;</string>
   </property>
   <property name="text">
    <string>   请在司机室占有条件下，执行整车火灾重启指令！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
